plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -852,6 +896,22 @@ function MosaicMenu:_recalculateDimen()
             height = corner_mark_size,
         }
     end
+
+    -- Create or replace progress_widget if needed
+    local progress_bar_width = self.item_width * 0.60;
+
+    if progress_widget == nil or progress_widget.width ~= progress_bar_width then
```

Comment:
`     if not progress_widget or progress_widget.width ~= progress_bar_width then`